source/features/fork-source-link-same-view.tsx

```diff
@@ -0,0 +1,64 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import * as api from '../github-helpers/api';
+import GitHubURL from '../github-helpers/github-url';
+
+const checkIfFileExists = async (url: GitHubURL): Promise<boolean> => {
+	const {repository} = await api.v4(`
+		repository(owner: "${url.user}", name: "${url.repository}") {
+			file: object(expression: "${url.branch}:${url.filePath}") {
+				... on Blob {
+					id
+				}
```

Comment:
Can this be simplified? We don't really need any details, so the shortest solution wins.

e.g.
```suggestion
				id
```